.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pywizlight/definitions"]
+	path = pywizlight/definitions
+	url = https://gitlab.com/eutampieri/wiz-definitions.git
```

Comment:
I see the benefit of having the bulb data in a reuseable format.

To be honest, this information should not live in a different repo on a different platform. This will add extra burden for people who wants to contribute (e. g., they need an additional account, they have to open PR/issue there and it will become hard to link/track changes).

Also, this will comes is a maintenance issue sooner or later. E. g., if one has no time to apply a fix then it require patching.

From a packaging point of view this will work as it does for now.

Reply:
Well, I’m trying to add support to a nodejs library so it would be great if the data was in an external repo so it can be easily depended on

Reply:
Also, if you are touching both repo in a contribution then it’s quite big already and I see the benefit of it being splitted